docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Database Access with Cassandra and ScyllaDB
```

Comment:
```suggestion
title: Database Access with AWS Keyspaces (Apache Cassandra)
```

A few other places in this guide mention "Cassandra and ScyllaDB" instead of "AWS Keyspaces"). Should we change it to match the guide link/content?